integreat_cms/cms/constants/colors.py

```diff
@@ -29,3 +29,7 @@
     "#fabebe",
     "#ffd8b1",
 ]
+
+#: Instruction for deciding between outline and ghost button.
+#: Please use outline buttons for secondary buttons - thus buttons that still fulfill the main purpose of the shown view, just in secondary hierarchy.
+#: Please use ghost buttons for buttons that have a different purpose than the other buttons on the shown view
```

Comment:
Two problems:
1. This will not be shown in the generated html documentation, because the `#:` syntax only works as docstring for something specific (e.g. an attribute or a function), if you just place this as inline comment, it will probably never be read...
2. The placement does not really make sense, yes, this is somehow about colors, but only for graph colors and not related to UI design in any way. So I doubt devs will read this when wondering about general design questions. I would prefer it if we would find a separate place for this, e.g. a new documentation page "Design guidelines" or similar?

Reply:
I added the documentation as proper .rst files to the "basic concepts" section of our documentation. I hope that was the solution you had in mind :)